its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -181,14 +187,22 @@ public static String replaceLtsVersion(String version) {
   }
 
   public static void reset(Orchestrator orchestrator) {
-    orchestrator.getServer().newHttpCall("api/orchestrator/reset")
-      .setMethod(HttpMethod.POST)
+    // We add one day to ensure that today's entries are deleted.
+    Instant instant = Instant.now().plus(1, ChronoUnit.DAYS);
+
+    // The expected format is yyyy-MM-dd.
+    String currentDateTime = DateTimeFormatter.ISO_LOCAL_DATE
+      .withZone( ZoneId.of("UTC"))
+      .format(instant);
+
+    LOG.info("TEST SETUP: deleting projects analyzed before: " + currentDateTime);
+
+    orchestrator.getServer()
+      .newHttpCall("/api/projects/bulk_delete")
       .setAdminCredentials()
+      .setMethod(HttpMethod.POST)
+      .setParams("analyzedBefore", currentDateTime)
       .execute();
-    // api/orchestrator/reset will clear the license, so reinstall it
-    if (orchestrator.getDistribution().isActivateLicense()) {
-      orchestrator.activateLicense();
-    }
   }
 
   static WsClient newWsClient(Orchestrator orch) {
```

Comment:
Empty line before `createTempFolder`: There're some spaces that can be cleaned as we code.